# MODEL1208060000: MODEL1208060000

## Installation

Download this repository, and install with distutils

`python setup.py install`

Or, install using pip

`pip install git+https://github.com/biomodels/MODEL1208060000.git`

To install a specific version (in this example, from the 2014-09-16 BioModels release)

`pip install git+https://github.com/biomodels/MODEL1208060000.git@20140916`


# Model Notes


Hoppe2012 - Predicting changes in metabolic function using transcript profiles

Measuring metabolite concentrations, reaction fluxes, and enzyme activities on
large scale are tricky tasks in the study of cellular metabolism. Here, a
method that predicts activity changes of metabolic functions based on relative
transcript profiles, has been presented. It provides a ranked list of most
regulated functions. The method has been applied to TGF-beta treatment of
hepatocyte cultures. This stoichiometric model of the mouse hepatocyte is
based on a corrected and extended version of HepatoNet1.

This model is described in the article:

[ModeScore: A Method to Infer Changed Activity of Metabolic Function from
Transcript Profiles](http://identifiers.org/doi/10.4230/OASIcs.GCB.2012.1)

Andreas Hoppe and Hermann-Georg Holzhütter

German Conference on Bioinformatics 2012; Publ.13.09.2012

Abstract:

Genome-wide transcript profiles are often the only available quantitative data
for a particular perturbation of a cellular system and their interpretation
with respect to the metabolism is a major challenge in systems biology,
especially beyond on/off distinction of genes. We present a method that
predicts activity changes of metabolic functions by scoring reference flux
distributions based on relative transcript profiles, providing a ranked list
of most regulated functions. Then, for each metabolic function, the involved
genes are ranked upon how much they represent a specific regulation pattern.
Compared with the naïve pathway-based approach, the reference modes can be
chosen freely, and they represent full metabolic functions, thus, directly
provide testable hypotheses for the metabolic study. In conclusion, the novel
method provides promising functions for subsequent experimental elucidation
together with outstanding associated genes, solely based on transcript
profiles.

This model is hosted on [BioModels Database](http://www.ebi.ac.uk/biomodels)
and identified by:
[MODEL1208060000](http://www.ebi.ac.uk/biomodels/MODEL1208060000) .

To cite BioModels Database, please use: BioModels Database: An enhanced,
curated and annotated resource for published quantitative kinetic models.
PMID: [20587024](http://identifiers.org/pubmed/20587024) .

To the extent possible under law, all copyright and related or neighbouring
rights to this encoded model have been dedicated to the public domain
worldwide. Please refer to [CC0 Public Domain
Dedication>http://creativecommons.org/publicdomain/zero/1.0/] for more
information.


